gnmi_cli_py/py_gnmicli.py

```diff
@@ -127,7 +133,9 @@ def _create_parser():
                       'Target when establishing secure gRPC channel.',
                       required=False, action='store_true')
   parser.add_argument('-x', '--xpath', type=str, help='The gNMI path utilized'
-                      'in the GetRequest or Subscirbe', required=True)
+                      'in the GetRequest or Subscirbe')
+  parser.add_argument('--xpath_list', type=str, help='The gNMI path utilized'
```

Comment:
>--xpath_list [](http://example.com/codeflow?start=23&length=12)

Possible to reuse original `-x` with `nargs="+"` ?

Reply:
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/telemetry/test_telemetry.py#L91
Zain is using xpath in sonic-mgmt test, we don't want to break telemetry end2end test.